From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 20 May 2021 02:42:12 +0200
Subject: tests: Do not run screencast tests

They require to have a full running pipewire (that requires a system
dbus daemon) so, we can't easily get one up and running while building.

So let's disable this for now
---
 src/tests/meson.build | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/src/tests/meson.build b/src/tests/meson.build
index 6364fd5..840f7cf 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -620,17 +620,17 @@ test_cases += [
       'native-virtual-monitor.c',
     ],
   },
-  {
-    'name': 'native-screen-cast',
-    'suite': 'backends/native',
-    'sources': [
-      'native-screen-cast.c',
-    ],
-    'depends': [
-      screen_cast_client,
-      screen_cast_client_driver,
-    ],
-  },
+#  {
+#    'name': 'native-screen-cast',
+#    'suite': 'backends/native',
+#    'sources': [
+#      'native-screen-cast.c',
+#    ],
+#    'depends': [
+#      screen_cast_client,
+#      screen_cast_client_driver,
+#    ],
+#  },
   {
     'name': 'bezier',
     'suite': 'backends/native',
